readthedocs/builds/tasks.py

```diff
@@ -82,6 +82,24 @@ def route_for_task(self, task, args, kwargs, **__):
             )
             return project.build_queue
 
+        # Use last queue used by the default version for external versions
+        if version.type == EXTERNAL:
+            last_build_for_default_version = (
+                project.builds
+                .filter(version__slug=project.get_default_version())
```

Comment:
I wonder if we should filter `success=True` here? Though I guess that isn't really a good check, and might miss ones without a successful build on the default project. 

I wonder if what we really want is an attribute on the project that keeps track of what queue it should use? I don't know if there's a "correct" way of handling this for every case though.

Reply:
> I wonder if what we really want is an attribute on the project that keeps track of what queue it should use?

I thought about something like this as well but I wasn't sure it was a good idea. We could talk a little more about this and share our thoughts. I think this PR is fine for now, tho.

> I don't know if there's a "correct" way of handling this for every case though.

Yeah, :disappointed: